testing/trino-faulttolerant-tests/src/test/java/io/trino/faulttolerant/jdbc/BaseJdbcFailureRecoveryTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.faulttolerant.jdbc;
+
+import io.trino.faulttolerant.BaseFailureRecoveryTest;
+import io.trino.operator.RetryPolicy;
+import org.testng.SkipException;
+
+import java.util.List;
+
+import static io.trino.execution.FailureInjector.FAILURE_INJECTION_MESSAGE;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public abstract class BaseJdbcFailureRecoveryTest
+        extends BaseFailureRecoveryTest
+{
+    public BaseJdbcFailureRecoveryTest(RetryPolicy retryPolicy)
+    {
+        super(retryPolicy);
+    }
+
+    @Override
+    protected void createPartitionedLineitemTable(String tableName, List<String> columns, String partitionColumn)
+    {
+    }
+
+    @Override
+    public void testJoinDynamicFilteringDisabled()
+    {
+        assertThatThrownBy(super::testJoinDynamicFilteringDisabled)
+                .hasMessageContaining("Table 'postgresql.tpch.partitioned_lineitem' does not exist");
+        throw new SkipException("skipped");
+    }
+
+    @Override
+    public void testJoinDynamicFilteringEnabled()
+    {
+        assertThatThrownBy(super::testJoinDynamicFilteringEnabled)
+                .hasMessageContaining("Table 'postgresql.tpch.partitioned_lineitem' does not exist");
+        throw new SkipException("skipped");
+    }
+
+    @Override
+    public void testAnalyzeTable()
+    {
+        assertThatThrownBy(super::testAnalyzeTable).hasMessageMatching("This connector does not support analyze");
+        throw new SkipException("skipped");
+    }
+
+    @Override
+    public void testDelete()
+    {
+        assertThatThrownBy(super::testDelete).hasMessageContaining(FAILURE_INJECTION_MESSAGE);
```

Comment:
hmm. Why so? Shouldn't we get "delete not supported" or sth alike?

Reply:
@losipiuk i don't get an unsupported error for this test. Maybe i'm missing something?

Reply:
It looks like bug in testing or connector. It should be passing. Looking into that (in the background)